trans/models.py

```diff
@@ -171,6 +173,18 @@ class UserContest(models.Model):
     note = models.TextField(default='')
     extra_country1 = models.CharField(max_length=6, blank=True)
     extra_country2 = models.CharField(max_length=6, blank=True)
+    extra_country1_count = models.IntegerField(
+        default=0,
+        validators=[
+            MaxValueValidator(4),
```

Comment:
Can we remove this max validator? The max value should be actually = country's `num_of_contestants`, but it is better to validate that in the app code (see my other comment below)